priv/signs.json

```diff
@@ -6165,7 +6188,139 @@
     "source_config": [
       [
         {
-          "stop_id": "70210",
+          "stop_id": "70502",
+          "direction_id": 0,
+          "headway_direction_name": "Heath Street",
+          "routes": ["Green-E"],
+          "platform": null,
+          "terminal": false,
+          "announce_arriving": true,
+          "announce_boarding": false
+        }
+      ]
+    ]
+  },
+  {
+    "id": "lechmere_green_line_mezzanine",
+    "headway_group": "green_trunk",
+    "pa_ess_loc": "GLEC",
+    "read_loop_offset": 120,
+    "text_zone": "m",
+    "audio_zones": ["m"],
+    "type": "realtime",
+    "source_config": [
+      [
+        {
+          "stop_id": "70501",
+          "direction_id": 1,
+          "headway_direction_name": "Union Square",
+          "routes": ["Green-E"],
+          "platform": null,
+          "terminal": false,
+          "announce_arriving": true,
+          "announce_boarding": false
+        },
+        {
+          "stop_id": "70502",
+          "direction_id": 0,
+          "headway_direction_name": "Heath Street",
+          "routes": ["Green-E"],
+          "platform": null,
+          "terminal": false,
+          "announce_arriving": true,
+          "announce_boarding": false
+        }
+      ]
+    ]
+  },
+  {
+    "id": "union_sq_east",
+    "headway_group": "green_trunk",
+    "pa_ess_loc": "GUNS",
+    "read_loop_offset": 30,
+    "text_zone": "e",
+    "audio_zones": [],
+    "type": "realtime",
+    "source_config": [
+      [
+        {
+          "stop_id": "70503",
+          "direction_id": 0,
+          "headway_direction_name": "Heath Street",
+          "routes": ["Green-E"],
+          "platform": null,
+          "terminal": true,
+          "announce_arriving": false,
+          "announce_boarding": false
+        },
+        {
+          "stop_id": "70504",
+          "direction_id": 0,
+          "headway_direction_name": "Heath Street",
+          "routes": ["Green-E"],
+          "platform": null,
+          "terminal": true,
+          "announce_arriving": false,
+          "announce_boarding": false
+        }
+      ]
+    ]
+  },
+  {
+    "id": "union_sq_west",
+    "headway_group": "green_trunk",
+    "pa_ess_loc": "GUNS",
+    "read_loop_offset": 30,
```

Comment:
Should we also give these the "standard" read loop offsets (seems to be 90 for west, 120 for mezzanine)? Even if nothing will actually be read for now, it could avert future issues if we split up the audio zones later.

Reply:
Would the signs not be reading out the headways or anything else at all? I figured that it keeping them on the same loop makes sense to avoid over frequently reading out the same content since they're all showing the same thing, but definitely correct me if I'm misinterpreting how the read loops would be behaving here

Reply:
The way you currently have it set up, all the audio zones are associated with just one of the configs. The other configs aren't connected to any audio zones, so their "read loop" is irrelevant — they _have_ an offset, because the format of the config requires it, but can't play audio since they aren't connected to any audio zones. At least, that's my understanding of how the audio system works, and my interpretation of @losvedir's comments above, but I should caution that I'm not an expert on this.

Reply:
Oh yes, that makes total sense. I was just confusing myself. Since they don't have any audio zones set up, then there wouldn't be any audio playing for those zones :)